build.msbuild

```diff
@@ -13,5 +15,12 @@
         <Exec Command="npm version --no-git-tag-version --allow-same-version $(PackageVersion)" />
         <Exec Command="gulp" />
         <Exec Command="nuget pack $(NuspecPath) -Version $(PackageVersion)" />
+        <Message Text="Testing Webpack build..." />
+        <MakeDir Directories="$(NpmPackOutputDir)" />
+        <Exec Command="npm pack $(MSBuildThisFileDirectory)" WorkingDirectory="$(NpmPackOutputDir)" />
+        <Exec Command="npm init -y" WorkingDirectory="$(WebPackTestDir)" />
+        <Exec Command="npm install $(NpmPackOutputDir)jquery-validation-unobtrusive-$(PackageVersion).tgz --save-dev --force" WorkingDirectory="$(WebPackTestDir)" />
+        <Exec Command="npm install webpack webpack-cli --save-dev" WorkingDirectory="$(WebPackTestDir)" />
+        <Exec Command="npx webpack" WorkingDirectory="$(WebPackTestDir)" />
```

Comment:
With the changes in this PR and the current scripts in the repo, I am able to repro the error.

@Eilon could we have this repo run in AppVeyor and Travis? (probably AppVeyor alone should be enough)

Reply:
FWIW AppVeyor has support for [linux builds](https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/) now which can be setup as part of a build matrix.

Reply:
Templating should now be enabled w/ AppVeyor.

Reply:
Oops. Wrong repo.

Reply:
jquery-validation-unobtrusive is now enabled in AppVeyor.

Reply:
@Eilon aspnet/templating runs Appveyor already